rs/ic_os/os_tools/setupos_tool/src/main.rs

```diff
@@ -50,61 +51,56 @@ pub fn main() -> Result<()> {
 
     match opts.command {
         Some(Commands::GenerateNetworkConfig { output_directory }) => {
-            let config_map = config_map_from_path(Path::new(&opts.config)).context(format!(
-                "Failed to get config.ini settings for path: {}",
-                &opts.config
-            ))?;
-            eprintln!("Using config: {:?}", config_map);
-
-            let network_info = NetworkInfo::from_config_map(&config_map)?;
-            eprintln!("Network info config: {:?}", &network_info);
-
-            let deployment_settings = get_deployment_settings(Path::new(&opts.deployment_file))
-                .context(format!(
-                    "Failed to get deployment settings for file: {}",
-                    &opts.deployment_file
-                ))?;
-            eprintln!("Deployment config: {:?}", deployment_settings);
-
-            let mgmt_mac = resolve_mgmt_mac(deployment_settings.deployment.mgmt_mac)?;
-            let deployment_environment = deployment_settings.deployment.name.parse()?;
+            let setupos_config: SetupOSConfig =
+                deserialize_config(DEFAULT_SETUPOS_CONFIG_OBJECT_PATH)?;
```

Comment:
Maybe this should be made the default for a new field on the command structs. This would give us a chance to run this outside of SetupOS.